index.html

```diff
@@ -271,6 +272,50 @@ <h3 class="ab-me-title">Skills</h3>
           </div>
         </section>
       </div>
+      <footer>
+        <div id="contact-head">
+          <h2>Contact me</h2>
+          <p id="contactme-text">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form action="https://formspree.io/f/mgedwpkj" method="POST">
+          <fieldset>
+            <input
+              class="work-child"
+              type="text"
+              name="full_name"
+              placeholder="Full name"
+              maxlength="30"
+              required
+            />
+
+            <input
+              class="work-child"
+              type="email"
+              name="email"
+              placeholder="Enter your email"
+              required
+            />
+
```

Comment:
- [ ] ***CONGRATULATION*** :clap: p: your contact form is working properly. However one of the requirements says that you should stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding). Now compare your design on the right and the design of the template on the left of the image :point_down:
![Screenshot from 2021-12-01 00-15-40](https://user-images.githubusercontent.com/65068771/144137490-eb5e7cc5-015b-498f-9fd8-de39766569fa.png)
 Kindly follow the template design and improve your contact form.

Reply:
Hello, I have been trying to download the correct image for an hour but it seems to be impossible, do you know who could help me with this?